packages.ini

```diff
@@ -1336,6 +1336,9 @@ python_versions = <3.11
 [sentry-kafka-schemas==0.1.47]
 [sentry-kafka-schemas==0.1.48]
 
+[sentry-ophio==0.1.1]
+python_versions = >=3.10
```

Comment:
this only limits the _download_ portion and not the check portion.  so your library is still being  checked against 3.9 at the moment

since this is an internal library you should target python 3.9